### PR TITLE
According to the proposal of @wkotlarski some changes are pulled.

### PR DESCRIPTION
**Pull request by uukhas**
They include:
1) Fixes to CXXDiagrams.m (when Transpose gave some errors),
2) Changes to test/module.mk to include 2 new tests,
3) Two new tests test/*.meta,
4) And new, safer realization of NPointFunctions + WilsonCoeffs itself.

### CODE REVIEW

File: test/module.mk [hunk 1]
```diff
@@ -198,7 +198,8 @@ endif
 ifeq ($(WITH_MRSSM2),yes)
 TEST_SRC += \
 		$(DIR)/test_MRSSM2_gmm2.cpp \
-		$(DIR)/test_MRSSM2_l_to_lgamma.cpp
+		$(DIR)/test_MRSSM2_l_to_lgamma.cpp \
+		$(DIR)/test_MRSSM2_l_to_l_conversion.cpp
```
Comment (wkotlarski) [inline test/module.mk]: we shouldn't bring in conversion test

File: test/module.mk [hunk 2]
```diff
@@ -837,6 +855,8 @@ $(DIR)/test_MRSSM2_gmm2.x: $(LIBMRSSM2)
 
 $(DIR)/test_MRSSM2_l_to_lgamma.x: $(LIBMRSSM2)
 
+$(DIR)/test_MRSSM2_l_to_l_conversion.x: $(LIBMRSSM2)
+
```
Comment (wkotlarski) [inline test/module.mk]: again, conversion test

File: meta/FlexibleSUSY.m [hunk 3]
```diff
@@ -4493,7 +4494,10 @@ corresponding tadpole is real or imaginary (only in models with CP
                           {FileNameJoin[{cxxQFTTemplateDir, "context_base.hpp.in"}],
                            FileNameJoin[{cxxQFTOutputDir, FlexibleSUSY`FSModelName <> "_context_base.hpp"}]},
                           {FileNameJoin[{cxxQFTTemplateDir, "npointfunctions.hpp.in"}],
-                           FileNameJoin[{cxxQFTOutputDir, FlexibleSUSY`FSModelName <> "_npointfunctions.hpp"}]}};
+                           FileNameJoin[{cxxQFTOutputDir, FlexibleSUSY`FSModelName <> "_npointfunctions.hpp"}]},
+                          {FileNameJoin[{cxxQFTTemplateDir, "npointfunctions_wilsoncoeffs.hpp.in"}],
+                           FileNameJoin[{cxxQFTOutputDir, FlexibleSUSY`FSModelName <> "_npointfunctions_wilsoncoeffs.hpp"}]}
+                          };
```
Comment (wkotlarski) [inline meta/FlexibleSUSY.m]: are we bringing in the Wilson coefficient calculator in this pull request?

Comment (uukhas) [reply meta/FlexibleSUSY.m]: @wkotlarski, this module is the only way now to work with external fermions and (potentially) external vector bosons.

File: test/module.mk (hunk 1 above)
Comment (uukhas) [reply test/module.mk]: @wkotlarski , yes, you are right, I am removing it.

File: test/module.mk (hunk 2 above)
Comment (uukhas) [reply test/module.mk]: @wkotlarski, I am removing this too.

**Pull request merged**